shell/dialog/ExtensionCatalogInstallDialog.vue

```diff
@@ -77,9 +76,9 @@ const DEFAULT_REPO = {
 };
 
 const initialState = () => {
-  const deploymentValues = structuredClone(DEFAULT_DEPLOYMENT);
-  const serviceValues = structuredClone(DEFAULT_SERVICE);
-  const repoValues = structuredClone(DEFAULT_REPO);
+  const deploymentValues = JSON.parse(JSON.stringify(DEFAULT_DEPLOYMENT));
+  const serviceValues = JSON.parse(JSON.stringify(DEFAULT_SERVICE));
+  const repoValues = JSON.parse(JSON.stringify(DEFAULT_REPO));
```

Comment:
This should be fine to clone with json parse/stringify based on the structure of the objects that we are cloning, but what is the reason for moving away from `structuredClone()`? I think that it's important that we understand any limitations that we might be encountering in this scenario.

Reply:
It was because of the unit tests... I wasn't really able to mock it. Let me have another go at it